### DISCUSSION TITLE
Spelling mistakes

### DISCUSSION THREAD

**Discussion by chakrabortydeepro** [PanditProject/Website#10] [Help] [answered]
I'd request you to correct the spellings of two "social identifiers"

1. Rājānaka instead of Rājanaka
2. Dharmādhikārin instead of Dharmādhikāri

**Comment by ybronner** [comment, marked as answer]
I'll do so. Thanks for suggesting this Deepro

**Reply by ybronner** [reply]
@AmirSimantov can you please do the edits?

**Reply by AmirSimantov** [reply]
Done.
